recipes/museval/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "museval" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2e1334dd3367dea562c1902515f6d55e87dc3072cfc411c616ac6f0a48d31d5f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: noarch
+
+requirements:
+  host:
+    - python >=3.6,<3.8
```

Does it really not work with 3.8?
```suggestion
    - python >=3.6
```

---

It does work indeed despite the lack of official support in the project :)